README.md

```diff
@@ -2,6 +2,7 @@
 <h1>
   Hedera DEX UI Accelerator 
 </h1>
+<p>Live App: <b><a href="https://defi-ui.hedera.com">defi-dex</a></b></p>
```

Comment:
We were requested to add the link (https://defi-ui.hedera.com) to the application in the README